udata/core/dataset/forms.py

```diff
@@ -60,6 +61,12 @@ class CommunityResourceForm(BaseResourceForm):
     organization = fields.PublishAsField(_('Publish as'))
 
 
+def map_legacy_frequencies(form, field):
```

Comment:
Added but the Deprecation timeline is (too me) out of the scope of this PR because:
- it needs to be discussed
- it needs some documentation
- it might needs some tooling
So I only added a skeleton section in the documentation